packages/xo-web/src/common/utils.js

```diff
@@ -681,3 +682,8 @@ export const getDetachedBackupsOrSnapshots = (
 
   return detachedBackupsOrSnapshots
 }
+
+// ===================================================================
+
+export const conditionalTooltip = (component, tooltip) =>
```

Comment:
It can be created in a dedicated module in order to avoid having a heavy `utils`.

Reply:
- if we create a component like this:
   ```js
       export const ConditionalTooltip = ({ content, children }) =>
         content !== undefined 
         ? <Tooltip content={content}>{children}</Tooltip> 
         : <span>{children}</span>
  ```
It will be not correct to use it for the `SortedTable` column.
IMHO, we keep it in `utils`. What do you think?

Reply:
You let the same implementation and just move it to a dedicated module.